core/models/officeblock.py

```diff
@@ -21,3 +21,23 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class FloorNumber(models.Model):
```

Comment:
Let's call the model to `OfficeFloor` and rename its fields
- `floor_number` to `number`
- `office_block` to `block`

Have a look at this [article](https://steelkiwi.com/blog/best-practices-working-django-models-python/) to understand the need for these changes.

Reply:
Fixed